src/applications/gi/gi-entry.jsx

```diff
@@ -1,14 +1,29 @@
 import 'platform/polyfills';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import './sass/gi.scss';
-
+import '../gi-sandbox/sass/gi.scss';
 import startApp from 'platform/startup/router';
 import { buildRoutes } from './routes';
 import reducer from './reducers';
 import manifest from './manifest.json';
 
-startApp({
-  url: manifest.rootUrl,
-  routes: buildRoutes(),
-  reducer,
-  entryName: manifest.entryName,
-});
+import { buildRoutes as redesignedRoutes } from '../gi-sandbox/routes';
+import redesignedReducer from '../gi-sandbox/reducers';
+
+const redesignFlag = toggleValues(0).gi_sandbox_comparision_tool_toggle;
+
+startApp(
```

Comment:
Non-blocking, but I think a cleaner way of writing this would be like this:

```js
const page = redesignFlag
  ? { reducer: redesignReducer, routes: buildRedesignRoutes() }
  : { reducer, routes: buildOriginalRoutes() };

startApp({
  url: manifest.rootUrl,
  entryName: manifest.entryName,
  ...page,
});
```

Reply:
Sure, why not?

Reply:
Looks good pending tests running again. Ping me if I need to re-approve for whatever reason. I can't see an option to approve again.